spec/outputs/syslog_spec.rb

```diff
@@ -34,21 +34,28 @@
 
   context "rfc 3164 and udp by default" do
     let(:options) { {"host" => "foo", "port" => "123", "facility" => "kernel", "severity" => "emergency"} }
-    let(:output) { /^<0>#{RFC3164_DATE_TIME_REGEX} baz LOGSTASH\[-\]: bar\n/m }
+    let(:output) { /^<0>#{RFC3164_DATE_TIME_REGEX} baz LOGSTASH: bar\n/m }
 
     it_behaves_like "syslog output"
   end
 
   context "rfc 5424 and tcp" do
-    let(:options) { {"rfc" => "rfc5424", "protocol" => "tcp", "host" => "foo", "port" => "123", "facility" => "kernel", "severity" => "emergency"} }
+    let(:options) { {"rfc" => "rfc5424", "protocol" => "tcp", "host" => "foo", "port" => "123", "facility" => "kernel", "severity" => "emergency", "procid" => "-" } }
```

Comment:
This change makes the test pass, but it breaks for all users, which are using a RFC 5424 configuration.